<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderHistoryWindow</class>
 <widget class="QMainWindow" name="OrderHistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史订单记录</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_username">
      <property name="text">
       <string>用户：用户名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="search_box">
        <property name="placeholderText">
         <string>输入订单号或日期搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filter_combo">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按日期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按餐桌</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="order_table">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>订单号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>餐桌</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总金额</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结账时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>详情</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back_btn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
